test/e2e/specs/managing-links.test.js

```diff
@@ -2,15 +2,178 @@
  * Internal dependencies
  */
 import {
+	META_KEY,
 	clickBlockAppender,
+	getEditedPostContent,
 	newPost,
+	pressWithModifier,
 } from '../support/utils';
 
+/**
+ * The modifier keys needed to invoke a 'select the next word' keyboard shortcut.
+ *
+ * @type {string}
+ */
+const SELECT_WORD_MODIFIER_KEYS = process.platform === 'darwin' ? [ 'Shift', 'Alt' ] : [ 'Shift', 'Control' ];
+
 describe( 'Managing links', () => {
 	beforeEach( async () => {
 		await newPost();
 	} );
 
+	it( 'Creating a link by selecting text and clicking the Link UI', async () => {
```

Comment:
I think this should be something like this, so that it makes a nice sentence:
```
describe( 'Creating a link by selecting text and clicking the Link UI', () => {
  it( 'creates the link placeholder', async () => {
```

Reply:
I've rejigged the `describe()` and `it()` descriptions in e5ef740b0 to read like English sentences 🇬🇧